src/core_plugins/kibana/server/routes/api/scroll_search/index.js

```diff
@@ -26,7 +26,7 @@ export function scrollSearchApi(server) {
     handler: (req, reply) => {
       const { callWithRequest } = server.plugins.elasticsearch.getCluster('admin');
       const { scrollId } = req.payload;
-      return callWithRequest(req, 'scroll', { scrollId })
+      return callWithRequest(req, 'scroll', { scrollId, scroll: '1m' })
```

Comment:
Should we instead pull the scroll value of the payload, and default to '1m' if it isn't specified? This would allow other consumers to set different durations.

Reply:
This is a legacy API, I don't feel we should add features to it.

Reply:
It should be removed when we remove our dependency on the scroll functionality on the client-side.